src/client/components/SliderComponent/SliderComponent.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import ArrowsComponent from '../ArrowsComponent/ArrowsComponent';
+import SlideshowImageComponent from '../SlideshowImageComponent/SlideshowImageComponent';
+import SliderDotsComponent from '../SliderDotsComponent/SliderDotsComponent';
+import avatarImages from '../../assets/data/ArrayOfImages';
+import './SliderComponent.styles.css';
+
+class Slider extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentImageIndex: 0,
+      totalImages: avatarImages.length,
+    };
+  }
+
+  gotoPreviousSlide = () => {
```

Comment:
These methods should really be merged into one since they are both doing the same thing. To much duplication

Reply:
I will merge them both